spec/support/page_objects/trainees/training_details.rb

```diff
@@ -0,0 +1,15 @@
+module PageObjects
+  module Trainees
+    class TrainingDetails < PageObjects::BasePage
+      set_url "/trainees/{id}/training-details"
+      element :submit_button, "input[name='commit']"
+      element :start_date_day, "input[name='trainee[start_date(3i)]']"
+      element :start_date_month, "input[name='trainee[start_date(2i)]']"
+      element :start_date_year, "input[name='trainee[start_date(1i)]']"
```

Comment:
could we use ID attributes instead of `input[name...]`

Reply:
done